static/sass/styles.scss

```diff
@@ -1305,6 +1305,14 @@ $color-link-dark: #69c !default;
   width: 100%;
 }
 
+.u-background--fade {
+  background-color: rgba(0, 0, 0, 0.05);
+}
+
+.u-aspect-ratio--16-9 {
```

Comment:
Would you mind adding a comment linking this issue https://github.com/canonical/vanilla-framework/issues/4876? We should remember to remove once it finally makes it to Vanilla

Reply:
Sure, done. These 2 were added on the fly with Lyubo on a call.